stack.go

```diff
@@ -148,32 +155,47 @@ func (c Call) Format(s fmt.State, verb rune) {
 // PC returns the program counter for this call frame; multiple frames may
 // have the same PC value.
 func (c Call) PC() uintptr {
-	return c.pc
+	return c.pcs[1]
```

Comment:
I believe this will be wrong in Go 1.9. A safe approach is to return `c.frame().PC`